src/backend/marsha/core/tests/test_api_shared_live_media.py

```diff
@@ -429,50 +429,65 @@ def test_api_shared_live_media_read_detail_student_ready_to_show_and_signed_url_
+                        "haredlivemedia/7520c16b-5846-41ca-822b-52b446a96809/1638230400.pd"
+                        "f?response-content-disposition=attachment%3B+filename%3Dpython-st"
+                        "ructures.pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6L"
+                        "y9hYmMuY2xvdWRmcm9udC5uZXQvZDlkNzA0OWMtNWEzZi00MDcwLWE0OTQtZTZiZj"
+                        "BiZDhiOWZiLyoiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9"
+                        "jaFRpbWUiOjE2MzgyMzc2MDB9fX1dfQ__&Signature=IVWMFfS7WQVTKLZl~gKgG"
+                        "ES~BS~wVLBIOncSE6yVgg9zIrEI1Epq3AVkOsI7z10dyjgInNbPviArnxmlV~DQeN"
+                        "-ykgEWmGy7aT4lRCx61oXuHFtNkq8Qx-we~UY87mZ4~UTqmM~JVuuLduMiRQB-I3X"
+                        "KaRQGRlsok5yGu0RhvLcZntVFp6QgYui3WtGvxSs2LjW0IakR1qepSDl9LXI-F2bg"
+                        "l9Vd1U9eapPBhhoD0okebXm7NGg9gUMLXlmUo-RvsrAzzEteKctPp0Xzkydk~tcnM"
+                        "kJs4jfbQxKrpyF~N9OuCRYCs68ONhHvypOYU3K-wQEoAFlERBLiaOzDZUzlyA__&K"
+                        "ey-Pair-Id=cloudfront-access-key-id"
                     ),
                     "pages": {
                         "1": (
                             "https://abc.cloudfront.net/d9d7049c-5a3f-4070-a494-e6bf0bd8b9"
                             "fb/sharedlivemedia/7520c16b-5846-41ca-822b-52b446a96809/16382"
-                            "30400_1.svg?Expires=1638237600&Signature=HhuTM-~v5kxNnQ4tPeI0"
-                            "yh8xvmJTwJa3AhedgDYWnpalIQcC~uEeU1yLO1gecVwzq4R0UaGf8rmbWvGwj"
-                            "cfG6T-5An7eRfK3JLz9-K7hRXSTFe-dyu9Xzy6vjP6WRj9qJWm7N7EIl8gj~I"
-                            "3E17IIALCfcXZ1oLYrrmKpikOZZfRUnzFKpiMVJ2yr0pM31T4rzjR-yiyupBp"
-                            "DfHomKsi8tS-b7D3Eymm9nKxjXK6zJ6sVE25kL1PejFUQnTQ55gNqsA5IhNuW"
-                            "rYOItpXHeFXoa9b2uGagyp1D3EPpv8p5fk5lizMq8fUykViloSWWR1DsQ6MND"
-                            "YU0L3TezfR4EUCMvQ__&Key-Pair-Id=cloudfront-access-key-id"
+                            "30400_1.svg?Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cH"
+                            "M6Ly9hYmMuY2xvdWRmcm9udC5uZXQvZDlkNzA0OWMtNWEzZi00MDcwLWE0OTQ"
+                            "tZTZiZjBiZDhiOWZiLyoiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7"
+                            "IkFXUzpFcG9jaFRpbWUiOjE2MzgyMzc2MDB9fX1dfQ__&Signature=IVWMFf"
+                            "S7WQVTKLZl~gKgGES~BS~wVLBIOncSE6yVgg9zIrEI1Epq3AVkOsI7z10dyjg"
+                            "InNbPviArnxmlV~DQeN-ykgEWmGy7aT4lRCx61oXuHFtNkq8Qx-we~UY87mZ4"
+                            "~UTqmM~JVuuLduMiRQB-I3XKaRQGRlsok5yGu0RhvLcZntVFp6QgYui3WtGvx"
+                            "Ss2LjW0IakR1qepSDl9LXI-F2bgl9Vd1U9eapPBhhoD0okebXm7NGg9gUMLXl"
+                            "mUo-RvsrAzzEteKctPp0Xzkydk~tcnMkJs4jfbQxKrpyF~N9OuCRYCs68ONhH"
+                            "vypOYU3K-wQEoAFlERBLiaOzDZUzlyA__&Key-Pair-Id=cloudfront-acce"
+                            "ss-key-id"
```

Comment:
IMO, we should extract the signature to a variable.
This way, we clearly show that the same one is used for all resources, and the test would be way more maintainable and clear.
WDTY ?

Let me know if you want to split this tedious work between you and me.

Reply:
Yes that's a good idea to extract the signature part :+1: 
I will do it, thanks !